mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,24 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-  return a + b + c;
+function addNumbers(a, b, c) {
+  let total = a + b + c;
+  return total;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+console.log(addNumbers(3, 4, 6));
 
-function getTotal(a, b) {
-  total = a ++ b;
+function introduceMe(name, age) {
+  return "Hello, my name is " + name + " and I am " + age + " years old";
```

Comment:
just as side note, it is also possible to use interpolation if that is more practical
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals